.circleci/config.yml

```diff
@@ -0,0 +1,64 @@
+---
+version: 2.1
```

Comment:
Most of our own projects, including Grafana, Loki and Tanka switched to from CircleCI to Drone. Jsonnet-Bundler also uses Drone.

No strong preference here, but might be worth to be consistent here

Reply:
Sounds good. But also happy to go forward with this.

Reply:
Lets go with Circle for now, as its done.  We can move to drone in the future.